docs/docs/sourcing-from-ghost.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Sourcing from Ghost
+---
+
+[Ghost](https://ghost.org) is an open source, professional publishing platform built on a modern Node.js technology stack — designed for teams who need power, flexibility and performance - used by Apple, NASA, Sky News, OpenAI & many more.
+
+It comes with all the benefits of modern, centralised Headless CMS platforms, with the added benefit of being released completely for free under an MIT license, so you have total ownership and control of it without needing to depend on a third party back-end.
+
+This guide will walk you through using [Gatsby](https://www.gatsbyjs.org/) with the [Ghost Content API](https://docs.ghost.org/api/content/).
+
+&nbsp;
+
+---
+
+## Quick start
+
+The fastest way to get started is with the official **Gatsby Starter Ghost** repository, which contains a light scaffolding of queries and templates to get a brand new site up and running.
+
+- Repository: https://github.com/tryghost/gatsby-starter-ghost
+- Demo: https://gatsby.ghost.org
+
+[![Gatsby Starter Ghost](./images/gatsby-starter-ghost.jpg)](https://gatsby.ghost.org)
+
+&nbsp;
+
+---
+
+## Install & setup
+
+If you prefer to start from scratch or integrate the Ghost Content API into an existing site, you can set up the **Gatsby Source Ghost** plugin.
+
+```shell
+npm install --save gatsby-source-ghost
+```
+
+### Configuration
+
+```javascript:title=gatsby-config.js
+// These are working demo credentials, try them out!
+
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-source-ghost`,
+      options: {
+        apiUrl: `https://gatsby.ghost.io`,
+        contentApiKey: `9cc5c67c358edfdd81455149d0`,
+      },
+    },
+  ],
+}
+```
+
+&nbsp;
+
+---
+
+## Generating pages
+
+Once the source plugin is set up, you can use the `createPages` API in `gatsby-node.js` to create queries on your Ghost data with GraphQL. In this example, Gatsby iterates over each post returned by the Ghost API and generates a new page with that data, using the `post.js` template file.
+
+There are several ways to structure queries depending on how you prefer to work, but here's a very minimal example:
+
+```javascript:title=gatsby-node.js
+const createPosts = new Promise((resolve, reject) => {
```

Comment:
This looks incomplete — did you intend this to be within a `exports.createPages = () =>` call?

https://www.gatsbyjs.org/docs/node-apis/#createPages